config.rb

```diff
@@ -18,6 +24,37 @@
 tag = JSON.parse(response)['tag_name']
 set :latest_release_tag, tag
 
+# Populate docs pages
+docs_source_dir = 'kubeless-src'
+checkout_commit = '1c592cdc55bf009dc58773a042e42d70085e930b' # Use tag or a specific commit if necessary
+system "cd #{docs_source_dir} && git fetch origin && git checkout #{checkout_commit}"
```

Comment:
Same here. Please don't call git from ruby. We don't know the environment where this code will be executed